cypress/support/fragments/inventory/inventorySearch.js

```diff
@@ -1,18 +1,52 @@
-import { MultiColumnList, MultiColumnListCell, MultiSelect, Pane } from '../../../../interactors';
+import { MultiColumnList, MultiColumnListCell, Pane, Accordion, Checkbox, TextField } from '../../../../interactors';
+import InventoryActions from './inventoryActions';
 
 
-const effectiveLocationInput = MultiSelect({ id: 'multiselect-6' });
+const effectiveLocationInput = Accordion({ id: 'effectiveLocation' });
+const languageInput = Accordion({ id: 'language' });
+const keywordInput = TextField({ id: 'input-inventory-search' });
 
 export default {
   effectiveLocation: {
-    // add new effective location inputs as needed
-    mainLibrary: 'Main Library'
+    mainLibrary: { id: 'clickable-filter-effectiveLocation-main-library' }
+  },
+  language: {
+    eng: { id: 'clickable-filter-language-english' }
   },
   getAllSearchResults: () => MultiColumnList(),
   getSearchResult: (row, col) => MultiColumnListCell({ 'row': row ?? 0, 'columnIndex': col ?? 0 }),
 
-  searchByEffectiveLocation(values) {
-    return effectiveLocationInput.select(values ?? [this.effectiveLocation.mainLibrary]);
+  selectResultCheckboxes(count) {
+    const clickActions = [];
+    for (let i = 0; i < count; i++) {
+      clickActions.push(this.getSearchResult(i).find(Checkbox()).click());
+    }
+    return cy.do(clickActions);
+  },
+
+  byEffectiveLocation(values) {
+    return cy.do([
+      effectiveLocationInput.clickHeader(),
+      effectiveLocationInput.find(Checkbox(values ?? this.effectiveLocation.mainLibrary)).click()
+    ]);
+  },
+
+  byLanguage(lang) {
```

Comment:
looks like lang is interactor filter. Need to add jsdoc in such not simple fragments places